### DISCUSSION TITLE
Problem with development

### DISCUSSION THREAD

**Discussion by peeyee** [dbeaver/dbeaver#21143] [Q&A]
Hi, guys! IF I want to develop a plugin, how to build and run dbeaver, Is there any docs? thanks.

**Comment by LonwoLonwo** [comment]
Hello @peeyee 

You can read more about developing in our WIKI:

https://github.com/dbeaver/dbeaver/wiki/Develop-in-Eclipse
https://github.com/dbeaver/dbeaver/wiki/Develop-in-IDEA
or https://github.com/dbeaver/dbeaver-workspace-vscode

Or if you want to build from sources: https://github.com/dbeaver/dbeaver/wiki/Build-from-sources
